account_reconcile_model_multicompany_propagate/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,3 @@
+To configure this module, you need to:
+
+#. Install module account that lets you manage reconcile models (if not already installed).
```

Comment:
Not really. This is a dependency, so it will be installed 100% of the times.

However, you have to get permissions to manage those, as you did in your video. Maybe you can explain that here instead.

You can also explain that you need to be granted access to other companies that share the same CoA.